scripts/cert-gen.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
Can we align the naming of scripts? Either with or without _.sh_ at the end. Personally I can go with both with a slight preference for with _.sh_. Either way it would be nice to be consistent.